mne/io/tests/test_reference.py

```diff
@@ -369,16 +369,17 @@ def test_set_bipolar_reference(inst_type):
 
     # Check channel information
     bp_info = reref.info['chs'][reref.ch_names.index('bipolar')]
-    an_info = reref.info['chs'][inst.ch_names.index('EEG 001')]
+    an_info = inst.info['chs'][inst.ch_names.index('EEG 001')]
     for key in bp_info:
-        if key == 'loc':
-            assert_array_equal(bp_info[key], 0)
-        elif key == 'coil_type':
-            assert_equal(bp_info[key], FIFF.FIFFV_COIL_EEG_BIPOLAR)
+        if key == 'coil_type':
+            assert_equal(bp_info[key], FIFF.FIFFV_COIL_EEG_BIPOLAR,
+                         err_msg=f'Key={key}')
```

Comment:
Nowadays we can just use plain `assert` so it's good to get in the habit for simple things like this:
```suggestion
            assert bp_info[key] == FIFF.FIFFV_COIL_EEG_BIPOLAR, key
```